src/service/profile.js

```diff
@@ -112,6 +113,7 @@ export const followUser = async (data) =>
         resolve(res.data);
       })
       .catch((err) => {
+        console.error(err.response);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure `err.response` exists before logging it to avoid potential errors.
```diff
- console.error(err.response);
+ if (err.response) {
+   console.error(err.response);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->